### DISCUSSION TITLE
why the returen data from search is not inaccurate

### DISCUSSION THREAD

**Discussion by othmanjad** [milvus-io/milvus#26948] [Q&A and General discussion]
Hello every one 

im trying a create text similrte search using this example 
https://milvus.io/docs/integrate_with_sentencetransformers.md
and I'm using the this csv file 
https://www.mediafire.com/file/7rkxayjxxbnsguq/names.csv/file
I'm trying to search by first name 

First Name: Robert Kendrick
Search Time: 0.00800013542175293
Results:
SHILONGO ---- 1.4181735515594482
AKINTOLA ---- 1.4181735515594482
MAIWADA ---- 1.4181735515594482
BROWN ---- 1.4181735515594482
SARKER ---- 1.4181735515594482
OLANIPEKUN ---- 1.4181735515594482
TSAFE ---- 1.4181735515594482

please note that i have a first name with Robert Kendrick

and if i have a name is abdullah
i need if the write a search text is abdallah
return in the result Abdullah

any one can help me

**Comment by yhmo** [comment]
I picked some items from the csv:

UID|lastname|firstname
1077950|MACFARLANE|Robert Kendrick
7381469|ABDULLAH|Mohammad Sharifudin Bin
1055699|SHILONGO|Festus
1063736|AKINTOLA|Rachael Boyede

Which column is converted to embeddings?
If you converted the "firstname" column to embeddings, then you can search the similar names in this way:
```
embeds = transformer.encode(["Robert"])

res = collection.search(
    data=embeds
    anns_field="embedding",
    param={},
    limit = 5,
    output_fields=['lastname']  # Include title field in result
)
```
It is supposed to return the "1077950|MACFARLANE|Robert Kendrick" as the most similar item for you.
